kernel/kernel-api/src/main/java/io/delta/kernel/Scan.java

```diff
@@ -48,7 +49,10 @@
 
 /**
  * Represents a scan of a Delta table.
+ *
+ * @since 3.0.0
```

Comment:
Sorry, this wasn't totally clear to me in your PR description.

Why is this a comment? `* @since 3.0.0` instead of an annotation like `@Evolving` below?

Reply:
`@since` is a tag supported by javadoc and `evolving` is not.

Reply:
Thanks!